pandas/tests/series/test_reshape.py

```diff
@@ -0,0 +1,59 @@
+import pytest
```

Comment:
well I did not find test for `series.unstack`, so just created one, pls let me know where the tests are stored, then I could move this part into the corresponding file

Reply:
See `tests/series/test_analytics.py`:

https://github.com/pandas-dev/pandas/blob/4e2546d89260fda592332a3988573c26edc7152c/pandas/tests/series/test_analytics.py#L163

Reply:
ahh! many thanks! @jschendel 

not sure if it is the best place for tests for `unstack` since `analytics` sounds a bit weird to `unstack`, maybe worth a follow-up to move to somewhere else @jreback

Reply:
yeah don't appear to be many, could create (and move the existing ones from test_analytics) to pandas/tests/series/test_reshaping.py to mirror frame